spec/factories/tweets.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :tweet do |tweet|
+    tweet.text "This is a Tweet."
```

Comment:
i don't mind hard coding text like this; some people prefer using the Faker gem.  sometimes hard coding things in factories can cause you problems, so just be aware of that.  :)